.pre-commit-config.yaml

```diff
@@ -26,33 +26,28 @@ repos:
         args: ['--autofix', '--no-sort-keys', '--indent=4']
       - id: end-of-file-fixer
       - id: mixed-line-ending
-  - repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: v0.3.5
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.6.9
     hooks:
     -   id: ruff
         args:
         - --fix
 
   - repo: https://github.com/asottile/pyupgrade
-    rev: v3.3.1
+    rev: v3.18.0
     hooks:
       - id: pyupgrade
-        args: [--py37-plus]
-        name: Upgrade code excluding monai networks
+        args: [--py39-plus, --keep-runtime-typing]
+        name: Upgrade code with exceptions
         exclude: |
           (?x)(
               ^versioneer.py|
-              ^monai/_version.py|
-              ^monai/networks/|  # no PEP 604 for torchscript tensorrt
```

Comment:
Will revert this two lines help?

Reply:
No. There was previously two runs of upgrade specified in the config yaml where the first exclude those files and then the second included only those files with the “keep-runtime-typing” flag used. After observing the same issues I attempted removing the exception and running pyupgrade once with “keep-runtime-typing” applied for all files.